packages/main/src/MenuSeparator.ts

```diff
@@ -0,0 +1,53 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import menuSeparatorTemplate from "./generated/templates/MenuSeparatorTemplate.lit.js";
+import menuSeparatorCss from "./generated/themes/MenuSeparator.css.js";
+import ListItem from "./ListItem.js";
+import { IMenuItem } from "./Menu.js";
+
+/**
+ * @class
+ * The `ui5-menu-separator` represents a horizontal line to separate menu items inside a `ui5-menu`.
+ * @constructor
+ * @extends UI5Element
+ * @abstract
+ * @public
```

Comment:
add `@implements` jsdoc tag to the class.

remove `@abstract` tag since this class has its own template.

change `@extends` to ListItem